festim/generic_simulation.py

```diff
@@ -503,7 +503,12 @@ def run_post_processing(self):
         self.update_post_processing_solutions()
 
         self.exports.t = self.t
-        self.exports.write(self.label_to_function, self.mesh.dx)
+        self.exports.write(
+            self.label_to_function,
+            self.mesh.dx,
+            self.materials,
+            self.settings.chemical_pot,
```

Comment:
I'm always careful when adding parameters like this because if we keep adding them then these functions because very hard to manage...